.github/workflows/check-contribution-form-filled.yml

```diff
@@ -3,6 +3,9 @@ on:
   pull_request:
     types: [opened, synchronize, labeled, unlabeled]
 
+permissions:
+  pull-requests: read
```

Comment:
Not sure this is needed. I think this is included in the default `read` permissions provided to workflow.

Reply:
Sorry ignore. Leave this as is.